recipe/meta.yaml

```diff
@@ -24,14 +24,14 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
-    - cmake >=3.18
+    - cmake >=3.23.1
     - make
   host:
-    - spdlog
-    - thrust >=1.9.0
+    - spdlog >=1.8.5,<1.9
```

Comment:
Even though the upstream recipe has this it isn't strictly required and this will break compatibility with the conda-forge pinning. I would suggest just leaving this unpinned so it gets migrated and whatnot.

Reply:
Thanks, Keith, fixed in 7345599.